src/wp-includes/script-loader.php

```diff
@@ -2872,12 +2872,18 @@ function wp_maybe_inline_styles() {
 
 	// Build an array of styles that have a path defined.
 	foreach ( $wp_styles->queue as $handle ) {
-		if ( wp_styles()->get_data( $handle, 'path' ) && file_exists( $wp_styles->registered[ $handle ]->extra['path'] ) ) {
+		$src  = $wp_styles->registered[ $handle ]->src;
+		$path = wp_styles()->get_data( $handle, 'path' );
+		if ( $path && $src ) {
+			$size = wp_filesize( $path );
```

Comment:
`wp_filesize` does both a `file_exists` and `filesize` check, so it's smart to combine these in one place. There is potential that the use of the `pre_wp_filesize` filter would skip the file_exists check, which would cause an `E_WARNING` level error below when `file_get_contents` is called. We might also want to add a check for a falsey value after `file_get_contents` is called below and continue the loop if there are no contents.

I like storing the size data to each handle so it's not recalculated. Nice idea!

Reply:
I pushed a change. Basically means that if src is empty to skip. This works before on the first run, it set to the src to false and second run it is already set to false. This means that second run, does not need to either calculate file sizes.